public/index.js

```diff
@@ -61,184 +48,9 @@ async function createWebSocket (loc) {
     var newBlock = JSON.parse(event)
     var b = newBlock.block
     b.unixStamp = (new Date(b.time)).getTime() / 1000
```

Comment:
This line can either be removed or changed to:
`newBlock.block.unixStamp = (new Date(b.time)).getTime() / 1000`

I'm not completely certain how this object is scoped so if it doesn't carry over to the new functions you added then it needs to be added to those functions instead.  Alternatively it could be changed to:
`newBlock.block.time = (new Date(b.time)).getTime() / 1000`
and just overwrite the timestamp at this point because it doesn't seem to get used as anything but a Unix timestamp.  Then I could change the go daemon to return the Unix timestamp instead of the formatted date that doesn't get used.  In which case the format function would need that multiply by 1000 added back in.

Reply:
I'm testing my proposed changes and so far everything looks good.  The newBlock object is scoped as I expected and `unixStamp` can be replaced by `time`.

Reply:
The formatted time string from the server's `TimeDef` type has direct compatibility with `Date`, so I'd like to leave `block.time` and `block.unixStamp` as separate representations. It doesn't add any appreciable bloat and they both have their uses right now. Further modification of time types would be far-reaching and outside the scope of this PR, but you could roll your own. I would suggest renaming `block.time` to `block.timeDef` or similar to add some clarity. 

There were also some big changes in the time types in #791 not long ago, so there seems to be a tug of war forming and I'm not picking sides yet.

Reply:
> 
> There were also some big changes in the time types in #791 not long ago, so there seems to be a tug of war forming and I'm not taking a side yet.

Yeah, I'm trying to decide how to resolve it.  As long as you calculate the unix timestamp on this line, then most of the times should kick back in.